cpp/src/hash/hashing.cu

```diff
@@ -679,7 +684,11 @@ std::unique_ptr<column> md5_hash(table_view const& input,
   auto chars_view = chars_column->mutable_view();
   auto d_chars    = chars_view.data<char>();
 
-  rmm::device_buffer null_mask{0, stream, mr};
+  rmm::device_buffer null_mask;
+  if (policy == retain_nulls::RETAIN_NULLS && input.num_columns() == 1)
+    null_mask = bitmask_and(input, mr, stream);
+  else
+    null_mask = rmm::device_buffer{0, stream, mr};
```

Comment:
```suggestion
  rmm::device_buffer null_mask = [&](){
  if (policy == retain_nulls::RETAIN_NULLS && input.num_columns() == 1)
   return bitmask_and(input, mr, stream);
  else
    return rmm::device_buffer{0, stream, mr};
   }();
```

Reply:
I don't understand this logic. It's calling `bitmask_and`, but only when there is a single column in the input?

Reply:
Sorry, that was a fragment from when I was sketching out what alternate null policies would look like. Removing anyway